addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -135,6 +138,7 @@ function changePaddingSize(direction) {
     }
     return {
         trigger: `.oe_overlay.ui-draggable.o_we_overlay_sticky.oe_active .o_handle.${paddingDirection}`,
+        extra_trigger: "body.editor_enable",
```

Comment:
useless extra trigger, you cannot  have .oe_overlay.ui-draggable.o_we_overlay_sticky withtout editor enable

Reply:
I thought this step also appeared when not in edit mode but this is indeed not the case.